src/components/HistoricRatesItem.js

```diff
@@ -21,6 +21,7 @@ export default class HistoricRatesItem extends Component {
 
   render() {
     const { ratesPerDate, date } = this.state;
+    const sizedRates = trimRates(ratesPerDate);
```

Comment:
Is it possible to use a different variable name.

Reply:
You'd have to be specific. A different variable name for which?

Reply:
For `sizedRates`.

Reply:
Okay.